tuf/api/metadata.py

```diff
@@ -1660,15 +1668,15 @@ class Targets(Signed):
     # pylint: disable=too-many-arguments
     def __init__(
         self,
-        version: int,
-        spec_version: str,
-        expires: datetime,
-        targets: Dict[str, TargetFile],
+        version: Optional[int] = None,
+        spec_version: Optional[str] = None,
+        expires: Optional[datetime] = None,
+        targets: Optional[Dict[str, TargetFile]] = None,
         delegations: Optional[Delegations] = None,
         unrecognized_fields: Optional[Mapping[str, Any]] = None,
     ) -> None:
         super().__init__(version, spec_version, expires, unrecognized_fields)
-        self.targets = targets
+        self.targets = targets or {}
```

Comment:
Do you think it would be a good idea to check for `None` again instead of a truthy value the same way as you suggested about `signatures`?
Maybe a user could want to just pass `{}` targets?

Reply:
yeah all these constructs are a bit suspect when the field is a container, checking for None is probably best